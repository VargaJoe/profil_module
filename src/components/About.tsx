import * as React				from 'react';

class About extends React.Component<any, any> {
    render() {

		return (
			<div className="about">
				{this.props.about}
			</div>
		);
	}
}

export default About;
 