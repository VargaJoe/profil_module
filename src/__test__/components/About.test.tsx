import * as React 						  	from 'react';
import About 							    from '../../components/About';
import { 
	configure, 
		shallow,
		mount } 							from 'enzyme';
import * as Adapter 					  	from 'enzyme-adapter-react-16';
import toJson 				      			from 'enzyme-to-json';

configure( {adapter: new Adapter()} );

describe('<About /> shallow rendering', () => {
		
	const about = mount(<About />); 
		
	// test Snapshot 
	it('Match to snapshot', () => {
		expect(toJson(about)).toMatchSnapshot();
	});

	it('Contain one text element ', () => {
		expect(about.find('.about').length).toBe(1);
	});
		 
});
 