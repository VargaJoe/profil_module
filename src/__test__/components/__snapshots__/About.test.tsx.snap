// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> shallow rendering Match to snapshot 1`] = `
<About>
  <div
    className="about"
  />
</About>
`;
